src/__tests__/decide.js

```diff
@@ -20,6 +20,27 @@ const expectDecodedSendRequest = (send_request, data, noCompression) => {
     })
 }
 
+const checkScriptsForSrc = (src, negate = false) => {
+    const scripts = document.querySelectorAll('body > script')
+    let foundScript = false
+    if (scripts.length > 0) {
```

Comment:
nit-picking... do we need the scripts length check?

MDN says `querySelectorAll` always returns a NodeList so it's safe to call length?

Reply:
oh, it's a test file in which case that's very nit picky 🤣

Reply:
oh lol, indeed good point I changed what's happening inside this condition later on, it's not needed anymore will remove xD